cmake-config

```diff
@@ -541,6 +541,7 @@ ${FAODEL_SYMBOLS} \
 -D ParMETIS_ROOT:PATH=${PARMETIS_PATH} \
 -D PNetCDF_ROOT:PATH=${PNETCDF_PATH} \
 -D fmt_ROOT:PATH=${FMT_PATH} \
+-D CMAKE_EXPORT_COMPILE_COMMANDS:BOOL=ON \
```

Comment:
Is this needed for something Catalyst uses, or just something that you turned on for something else and left in?

I do use that at times, and might be OK to always use enabled, but just checking so I know why you have it on.